### DISCUSSION TITLE
GetPropertyValueAsync

### DISCUSSION THREAD

**Discussion by sn0wcr4sh** [ChromiumDotNet/WebView2.DevTools.Dom#8] [General]
Hi, 
Value of GetPropertyValueAsync<T> seems to be always returning string, regardless of T. Is this on purpose/error or am I missing some point in how it should be used?  
Thanks

**Comment by amaitland** [comment]
Please provide example code.

https://github.com/ChromiumDotNet/WebView2.DevTools.Dom/blob/main/ISSUE_TEMPLATE.md#complete-minimal-example-reproducing-the-issue

**Comment by sn0wcr4sh** [comment]
Ok, let's say I want to get z-index, so

```c#
var div = await devToolsContext.QuerySelectorAsync<HtmlDivElement>("#myDiv");  
var style = await div.GetStyleAsync();  
var zindex = await style.GetPropertyValueAsync<int>("z-index");
```

So given the generic method ```GetPropertyValueAsync<T>```, I would expect that zindex will be int in my case

But it is string (at least for all properties I have tried so far)

**Reply by amaitland** [reply]
What is the value of the string?

Does the style actually have a z-index?

**Reply by amaitland** [reply]
I see the problem, please create a new issue.
